projects/step-frontend/src/lib/modules/timeseries/chart/time-series-chart.component.ts

```diff
@@ -164,7 +164,24 @@ export class TimeSeriesChartComponent implements OnInit, AfterViewInit, OnChange
         },
         ...settings.series,
       ],
-      hooks: {},
+      hooks: {
+        // init: [
```

Comment:
if something will be commented out and preserved, it would be better to provide a descriptive comment as to why it was not removed in the first place